scholia/app/templates/venue.html

```diff
@@ -261,6 +261,39 @@
 ORDER BY DESC(?year)
 LIMIT 500
  `
+ retractionsJournalSparql =  `
+SELECT DISTINCT
+  ?retracted_work ?retracted_workLabel
+  ?date
+  ?citing_work ?citing_workLabel
+WITH {
+  # Find retracted papers indicated by instance of retracted paper, 
+  # by retraction notice property or by significant event
+  SELECT DISTINCT ?retracted_work WHERE {
+    { ?retracted_work wdt:P31 wd:{{ q }} }
```

Comment:
I do not understand this line. `?retracted_work` becomes instance of work? Shouldn't it be instance of retracted work (if such an item exists)?

Reply:
I see we have wd:Q45182324

Reply:
Is this solved? Otherwise, Egon could you answer this?

Reply:
I think the line was wrong, also in revised version. It should read `?retracted_work wdt:P31 wd:Q45182324` (fix pending).